client/landing/stepper/declarative-flow/internals/steps-repository/woo-verify-email/index.tsx

```diff
@@ -1,30 +1,77 @@
+import { Button } from '@automattic/components';
 import { StepContainer } from '@automattic/onboarding';
+import { createElement, createInterpolateElement } from '@wordpress/element';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
+import { useSelector } from 'react-redux';
 import FormattedHeader from 'calypso/components/formatted-header';
+import { useSiteSlugParam } from 'calypso/landing/stepper/hooks/use-site-slug-param';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { UserData } from 'calypso/lib/user/user';
+import { getCurrentUser } from 'calypso/state/current-user/selectors';
 import type { Step } from '../../types';
+import './style.scss';
+
+function redirect( to: string ) {
```

Comment:
I know we've used this method elsewhere in the Stepper framework; should we make it a hook for easier use across steps/DRYer code?

Reply:
I will create an issue for that, ok?